package.json

```diff
@@ -23,15 +23,17 @@
     "build:server": "NODE_ENV=production webpack --config ./webpack/server.babel.js",
     "build:db": "node ./src/server/database/setupDB.babel.js",
     "update:db": "node ./src/server/database/setupDB.babel.js true",
-    "test": "NODE_ENV=testing ava ./src/**/__tests___/**/*.js --verbose",
```

Comment:
These should undoubtedly be separate. Pretty sure xo added `xo &&` to the front of `test` and I somewhat mindlessly replaced `xo` with `npm run lint`.